src/features/classification/Classification.js

```diff
@@ -19,12 +19,14 @@ type Props = {
     controls?: Controls,
     controlsFormat?: ControlsFormat,
     definition?: string,
+    fillColor?: string,
```

Comment:
Does it make sense to make it a color object if there is possibility of more color customization? Or if its just going to be these two, then I guess this is fine

Reply:
Also, if there is going to be specific colors only, then maybe the prop should be like theme="purple". And internally you map purple to the colors

Reply:
cc @feixuanli

Reply:
We're planning to have a color map but in this case I'm trying to keep the Classification components agnostic to it since it might be moved to a more central location (in order to make it available to mobile, desktop, etc.). Eventually de API itself could be returning the colors so we need the components to be flexible.

Reply:
But yes, we're not planning to support more colors beyond the fill and stroke

Reply:
Agreed. I ll include the color map in my PR with radio buttons.